app/dashboard/views.py

```diff
@@ -815,23 +815,39 @@ def users_fetch(request):
     all_users = []
     for user in all_pages.page(page):
         profile_json = {}
-        profile_json = user.to_standard_dict()
-        if user.avatar_baseavatar_related.exists():
-            user_avatar = user.avatar_baseavatar_related.first()
-            profile_json['avatar_id'] = user_avatar.pk
-            profile_json['avatar_url'] = user_avatar.avatar_url
+        # profile_json = user.to_standard_dict()
         count_work_completed = Activity.objects.filter(profile=user, activity_type='work_done').count()
         count_work_in_progress = Activity.objects.filter(profile=user, activity_type='start_work').count()
+        profile_json['id'] = user.id
+        profile_json['actions_count'] = user.actions_count
+        profile_json['created_on'] = user.created_on
+        profile_json['data'] = user.data
+        profile_json['email'] = user.email
+        profile_json['handle'] = user.handle
+        profile_json['hide_profile'] = user.hide_profile
+        profile_json['job_status'] = user.job_status_verbose if user.job_search_status else None
+        profile_json['show_job_status'] = user.show_job_status
+        profile_json['job_location'] = user.job_location
+        profile_json['job_salary'] = user.job_salary
+        profile_json['job_search_status'] = user.job_search_status
+        profile_json['job_type'] = user.job_type
+        profile_json['linkedin_url'] = user.linkedin_url
+        profile_json['resume'] = user.resume
+        profile_json['remote'] = user.remote
+        profile_json['keywords'] = user.keywords
+        profile_json['organizations'] = user.organizations
         profile_json['position_contributor'] = user.get_contributor_leaderboard_index()
         profile_json['position_funder'] = user.get_funder_leaderboard_index()
         profile_json['work_done'] = count_work_completed
         profile_json['work_inprogress'] = count_work_in_progress
-
-        profile_json['job_status'] = user.job_status_verbose if user.job_search_status else None
         profile_json['verification'] = user.get_my_verified_check
         profile_json['avg_rating'] = user.get_average_star_rating
```

Comment:
^ would it be cleaner to redefine the whole bit as 

```
profile_json = {
  id: user.id,
  ...
}
```

Reply:
@thelostone-mc actually is done here https://github.com/gitcoinco/web/pull/4477